crates/bevy_pbr/src/meshlet/asset.rs

```diff
@@ -25,8 +29,8 @@ use std::sync::Arc;
 /// See also [`super::MaterialMeshletMeshBundle`] and [`super::MeshletPlugin`].
 #[derive(Asset, TypePath, Serialize, Deserialize, Clone)]
 pub struct MeshletMesh {
-    /// The total amount of triangles summed across all meshlets in the mesh.
-    pub total_meshlet_triangles: u64,
+    /// The total amount of triangles summed across all LOD 0 meshlets in the mesh.
+    pub worst_case_meshlet_triangles: u64,
```

Comment:
Maybe `max_meshlet_triangles`?

Reply:
I personally feel that that's less clear. Up to maintainers.

Reply:
i prefer keeping it as is